btr-api/tests/unit/resources/test_submissions.py

```diff
@@ -936,6 +936,70 @@ def test_post_plots_email_error(app, client, session, jwt, requests_mock):
             assert created_submission.business_identifier == json_data['businessIdentifier']
 
 
+def test_post_plots_invalid_verification(app, client, session, jwt, requests_mock):
```

Comment:
I'd like to see a test that verifies it works with verificationStatus and/or fails without it if you make it a required field